### ISSUE TITLE
storage information

### ISSUE THREAD

**Issue by ferrighi** [steingod/mdharvest#37]
It should be possible to add some storage_information elements for traverse_thredds. 
The file format is particularly relevant for translation to ISO/Inspire.

**Comment by steingod** [comment]
Not sure what you mean by storage_information here?

**Comment by ferrighi** [comment]
I refer to https://htmlpreview.github.io/?https://github.com/metno/mmd/blob/master/doc/mmd-specification.html#storage_information 
So when creating mmd records we could populate this section.

**Comment by ferrighi** [comment]
Yes, the file format is the only element that is relevant for translation to gmd:MD_Format in ISO.

**Comment by steingod** [comment]
Good question, I just commented on the PR on this. Ideally we should have the NCML, onwards we will use NCML and not aggregation by configuration so should not be an issue onwards I hope.
